CHANGES.md

```diff
@@ -1,3 +1,24 @@
+## edge-19.6.4
+
+This release adds support for the SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+API.  Creating a TrafficSplit resource will cause Linkerd to split traffic
+between the specified backend services.  Please see [the spec](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+for more details.
+
+* CLI
+  * Added a check to install to prevent installing multiple control planes into
+    different namespaces
+  * Added support for passing a URL directly to linkerd inject
+  * Added a `--all-namespaces` flag to `linkerd edges`
+* Controller
+  * Added support for the SMI TrafficSplit API which allows users to define
+    traffic splits in TrafficSplit custom resources
+* Proxy
```

Comment:
Do we normally areas like "Proxy" when there are no updates? Should we add something about traffic split support in the proxy?

Reply:
good catch.  traffic split spans controller and proxy but I decided to put it in the controller section.  I don't think we need it in both.  I'll remove the empty proxy bullet.